whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java

```diff
@@ -44,6 +44,8 @@
 import java.util.Enumeration;
 import java.util.List;
 
+import static java.time.format.DateTimeFormatter.RFC_1123_DATE_TIME;
```

Comment:
Can you update the Formatter in the `MessageParser.DATE_FORMAT` field instead? The format of RFC_1123_DATE_TIME is different in more than the timezone.

Reply:
Or use the standard 'UTC' formatter we're using elsewhere

Reply:
This seems to be the format the smtp server uses for the `Delivery-date` time. And it is used when parsing from the `Delivery-date` string representation to time in Java